node/src/actors/chain_manager/actor.rs

```diff
@@ -61,7 +61,13 @@ impl ChainManager {
             })
             .and_then(|config, act, _ctx| {
                 let consensus_constants = config.consensus_constants.clone();
+
+                // Set ChainManager parameters related with the consensus constants
```

Comment:
Actually, consensus constants can be accessed as


    self
        .chain_state
        .chain_info
        .as_ref()
        .unwrap()
        .consensus_constants

So we don't need to store any of these